apps/csp/src/app/form-base/form-base.component.ts

```diff
@@ -215,10 +219,10 @@ export class FormBaseComponent implements OnInit, AfterViewInit {
     this.processErrors();
   }
 
-  // processRegulatoryInfoErrors(errorList) {
-  //   this._regulatoryInfoErrors = errorList;
-  //   this.processErrors();
-  // }
+  processTimmingOfApplicantErrors(errorList) {
```

Comment:
processTimming - can you remove the extra m in processTiming

Reply:
fixed